tests/sparseml/transformers/test_transformers.py

```diff
@@ -152,13 +179,15 @@ def test_outputs_ort(self, setup):
         path_onnx = model.onnx_file.downloaded_path()
         model_path = model.framework_files[0].dir_path
         inputs_path = model.data_inputs.path
+        _attempt_copy_original_recipe(model_path)
 
         input_data = load_numpy_list(inputs_path)[0]
 
         path_retrieved_onnx = export_transformer_to_onnx(
             task=task,
             model_path=model_path,
             onnx_file_name=self.onnx_retrieved_name,
+            sequence_length=next(iter(input_data.values())).shape[0],
```

Comment:
Oh shoot, it looks like you were not able to find the `sequence_length` in the attributes of the model.
Tbh, I really do not like the fact that one needs to go such an extra mile to retrieve it.
@bfineran how do you feel about adding `sequence_length` as an attribute of a transformer model (downloaded from stub)?

Reply:
From what I can tell, you need to load in the transformer model to get that attribute. The `model` shown here is a SpraseZoo model object, rather than a transformers model object. We probably don't want to add a `sequence_length` property to that class, but what might be a reasonable solution is to implement a function to determine sequence length within the transformers `export.py`, once the actual model is loaded.

Reply:
This is fine for now, so we can compare outputs potentially. Sequence length can be changed on compilation so is not necessarily tied to a particular model.